pkg/thing/mocks/connector.go

```diff
@@ -0,0 +1,27 @@
+package mocks
+
+import (
+	"github.com/CESARBR/knot-babeltower/pkg/thing/entities"
+	"github.com/stretchr/testify/mock"
+)
+
+type FakeConnector struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type FakeConnector should have comment or be unexported](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4853780)